about/pricing-and-account-management.md

```diff
@@ -133,49 +133,50 @@ The available plans are:
+| Cloud SQL editor                               | 3 seats                                 | 10 seats                       | 20 seats                            |
+| Charts                                                        | ✓                                       | ✓                              | ✓                                   |
+| Dashboards                                                    | 2                                       | Unlimited                      | Unlimited                           |
+| **Storage and performance**                                   |                                         |                                |                                     |
+| IOPS (autoscales)                                             | 	3,000 - 5,000	                         | 5,000 - 8,000                  | 5,000 - 8,000                       | 
+| Bandwidth (autoscales)	                                       | 125 - 250 Mbps                          | 	250 - 500 Mbps                | 	Up to 500 mbps                     | 
+| IO Boost	                                                     |                                         | 	Add-on: <br/>16K IOPS, 1000 Mbps BW	 | Add-on: <br/>16K IOPS, 1000 Mbps BW | 
+| **Availability and monitoring**                               |                                         |                                |                                     |
+| High-availability replicas <br/>(Automated multi-AZ failover) | ✓                                       | ✓                              | ✓                                   |
+| Read replicas		                                               |                                         | ✓                              | ✓                                   |
+| Point-in-time recovery and forking                            | 	3 days                                 | 14 days                        | 14 days                             |
+| Performance insights                                          | ✓                                       | ✓                              | ✓                                   |
+| Metrics and log exporters	                                    |                                         | ✓                              | ✓                                   |
+| **Security and compliance**                                   |                                         |                                |                                     |
+| End-to-end encryption                                         | ✓                                       | ✓                              | ✓                                   |
+| Private Networking (VPC)                                      | 1 multi-attach VPC	                     | Unlimited multi-attach VPCs    | 	Unlimited multi-attach VPCs        | 
+| IP address allow list | 1 allow list with up to 10 IP addresses | Unlimited                      | Unlimited |
```

Comment:
Correction: 10 allow lists with 10 addresses instead of "unlimited" for both enterprise and scale.
cc @esclymeTS

Reply:
I'm also curious why do we say "unlimited" for VPCs whereas we allow max 3 on scale/enterprise. It's possible to request more but it's a support ticket process

Reply:
Updated